### DISCUSSION TITLE
Frontend Migration

### DISCUSSION THREAD

**Discussion by Not-Sarthak** [cockpit-project/cockpit#19783] [General]
Hey! I was trying to understand multiple repos under the organization and came up to knowing that the frontend is HTML heavy. Can we initiate and slowly migrate the whole webapp to React/Next and Tailwind. 

Benefits: 
- Code Splitting
- SSR (Server-Side Rendering)
- Better Developer Experience
- Improved UI/UX 
- Maintainability

**Comment by martinpitt** [comment]
What do you mean with "HTML heavy"? The only thing that is is the login page, for good reasons. All the cockpit pages have more or less just a stub HTML page, the rest is already React.
